pkg/kafka/subscriber.go

```diff
@@ -560,6 +562,9 @@ ResendLoop:
 		case <-msg.Acked():
 			if sess != nil {
 				sess.MarkMessage(kafkaMsg, "")
+				if !h.saramaConfig.Consumer.Offsets.AutoCommit.Enable {
```

Comment:
Hey @donmikel! Thanks for the PR. Can you add a test for that case in that file: https://github.com/ThreeDotsLabs/watermill-kafka/blob/master/pkg/kafka/pubsub_test.go

I think that it is not needed to run entire `tests.TestPubSub`. I think that it should be enough to use [TestPublishSubscribe](https://github.com/ThreeDotsLabs/watermill/blob/e1fb767c92e58ff95d40d56e501ff9966a709426/pubsub/tests/test_pubsub.go#L204) from the test suite.